Data/AuthenticationClient/Sources/AppAuthOAuthService.swift

```diff
@@ -0,0 +1,158 @@
+//
```

Comment:
Since this is an implementation of Core/OAuthService. Can we have it under maybe Core/OAuthServiceImpl target?